AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/ASF/ASFUIDeviceInfo.swift

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import UIKit
```

Comment:
Wrap it around a `canImport`? Since we are moving to supporting macOS too..

Reply:
Added